Makefile

```diff
@@ -8,8 +8,12 @@ help:
 SHELLCHECK=shellcheck
 YAMLLINT=yamllint
 
+DEPLOY_ENV_MAX_LENGTH=15
+DEPLOY_ENV_VALID=$(shell if [ $${\#DEPLOY_ENV} -gt $(DEPLOY_ENV_MAX_LENGTH) ]; then echo ""; else echo "OK"; fi)
+
 check-env-vars:
 	$(if ${DEPLOY_ENV},,$(error Must pass DEPLOY_ENV=<name>))
+	$(if ${DEPLOY_ENV_VALID},,$(error Sorry, DEPLOY_ENV has a max length of 15, otherwise derived names will be too long))
```

Comment:
Instead of hardcoding '15' here, can it reference the above variable, otherwise this error message could get out of sync.